src/Differ.php

```diff
@@ -18,6 +18,12 @@
  */
 final class Differ
 {
+    const OLD                     = 0;
```

Comment:
These should be `private`, right?

Reply:
I thought about scoping these down, like using visibility `private` or  `@internal` annotations, but  these values should be available to the output builders

Reply:
Then please add an explicit `public` keyword. Thanks!

Reply:
Just realized, this package supports `"php": "^7.0"` and visibility modifiers for class constants were added in 7.1, sadly because I would've liked to use those here.